### ISSUE TITLE
No such file or directory: hairpinTmp.fa

### ISSUE THREAD

**Issue by vivekruhela** [mhalushka/miRge#34]
I am trying to run these pipelines on one of the public data ([here](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123436)) . This pipeline is working fine except at one sample which is [CL176](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSM3503237). I am not able to understand the reason for its failure at just particular sample. I have checked this sample and it is a valid fastq file. Here is the screenshot of the error.
![Screenshot from 2020-07-13 19-08-18](https://user-images.githubusercontent.com/13174913/87311732-42a4e080-c53d-11ea-84e2-42c2e8c5dc36.png)

Please suggest. Thanks.

**Comment by arunhpatil** [comment]
Hi Vivek, 

The same data (SRR8286233) was executed locally and it was successful without any errors. To better understand this issue, could you try the same command with `-d MirGeneDB` option? Let us know the status of this execution. 

Thank you,
Arun

**Comment by vivekruhela** [comment]
Thanks for your response. I have executed the other samples with miRBase database. This sample should also work with the same reference database of miRNA. Why change of reference database is required here? All the samples should be executed with the same reference otherwise results will not be justifiable. Have you tried with reference miRBase database and got success???

**Comment by arunhpatil** [comment]
Yes, you are right about justification. Yes, the sample was executed with miRBase and was successful in the local machine. To try and understand the problem, I want to see if it fails with MirGeneDB as well. Please try with MirGeneDB and let me know the status. Thanks.

**Comment by vivekruhela** [comment]
It is working. But it is taking too much time. I started this pipeline with the option `-d MirGeneDB` in the morning 8 hours before but it is still running. By the way, even after completion, these results are of no use to me because I want it for the miRBase database only. Can you suggest what are the issues coming and why? For time-saving, can you send me `mapped.csv` and `novel_miRNA_report.csv` for this sample (CL176) with the miRBase reference database? Thanks.

**Comment by vivekruhela** [comment]
UPDATE: miRge pipeline just finished now and failed. There were no results after 8 running hour. Now, it is confirmed that in my server mirge is not working for this sample.

**Comment by arunhpatil** [comment]
Hi Vivek, 

As per your request, please find the result files from SRR8286233 in this [link](https://livejohnshopkins-my.sharepoint.com/:f:/g/personal/ahanuma2_jh_edu/ErRrlthhuCdHtkBqXX4DAtEBxFuPFUZtBEch4VWBANFcbg?e=Rk3e0Y). I will get back to you with your recent status of miRge execution. 

Thank you,
Arun.

**Comment by vivekruhela** [comment]
Thanks for sharing.
